Ark.Tools.HealthCheck/Ark.Tools.AspNetCore.HealthCheck.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
duplicated file?

Reply:
Yes or copied csproj for nuget configurations . Is there something incorrect?

Reply:
this looks duplicated, there is both 
Ark.Tools.AspNetCore.HealthCheck/Ark.Tools.AspNetCore.HealthCheck.csproj
Ark.Tools.HealthCheck/Ark.Tools.AspNetCore.HealthCheck.csproj

the latter has project and folder that doesn't match.